cobalt/browser/cobalt_content_browser_client.cc

```diff
@@ -49,9 +44,7 @@
 #include "content/public/common/content_switch_dependent_feature_overrides.h"
 #include "content/public/common/user_agent.h"
 #include "content/shell/browser/shell.h"
-// TODO(b/390021478): Remove this include when CobaltBrowserMainParts stops
```

Comment:
nit: Could you leave the TODO but associated with l.46 plz? Something like ` //TODO(...): Remove Shell dependencies.

Reply:
This line was recently removed when I cleaned up the #includes.